src/main/java/BottleOpener/BottleOpener.java

```diff
@@ -0,0 +1,47 @@
+package BottleOpener;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+
+/**
+ * The {@code BottleOpener.BottleOpener} class is the main entry point for the chatbot.
+ * It initializes the necessary classes, loads the saved tasks, and processes user input
+ * until the user decides to exit the application.
+ */
+public class BottleOpener {
+
+    /**
+     * The main method of the BottleOpener.BottleOpener chatbot. It sets up the user interface, loads tasks from storage,
+     * and starts the command processing loop.
+     *
+     * @param args Command-line arguments (not used).
+     */
+    public static void main(String[] args) {
+        String botName = "BottleOpener";
+        String filepath = "data/BottleOpener.txt";
```

Comment:
Should change these into a constant instead since they are hard coded
```java
private final String BOT_NAME = "BottleOpener";
private final String FILE_PATH = "data/BottleOpener.txt";
```